src/campaign.js

```diff
@@ -144,6 +144,32 @@ const { Util } = require("./util.js");
           faultString = faultString.trim();
       }
 
+      // https://github.com/adobe/acc-js-sdk/issues/108
+      // 401 error code is hidden in error message and incorrectly reported as HTTP 500 error causing the refresh token
+      // callback not to be called
+      // Extract specific error code if possible
+      if (errorCode == "SOP-330007" && errorMessage && errorMessage.indexOf("XSV-350114") != -1 && errorMessage.indexOf(" 401") != -1) {
```

Comment:
I guess the space character before 401 is there on purpose, maybe we could encode it (`\u0020`) to prevent any unwanted removal.

Reply:
Optionally comparisons can be made strong with `===`